pkgs/development/python-modules/graph-tool/2.x.x.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, python, cairomm, sparsehash, pycairo, automake, m4, pkgconfig, boost, expat, scipy, numpy, cgal, gmp, mpfr, lndir, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  version = "2.2.36";
+  name = "${python.libPrefix}-graph-tool-${version}";
+
+  meta = with stdenv.lib; {
+    description = "Graph-tool is an efficient Python module for manipulation and statistical analysis of graphs (a.k.a. networks).";
```

Comment:
Please update `description` according to http://nixos.org/nixpkgs/manual/#idm140737319319584